### ISSUE TITLE
Display of Special Characters in Response Broken

### ISSUE THREAD

**Issue by tmack8001** [a85/POSTMan-Chrome-Extension#2800]
Version Information: 
Postman Version 4.10.2 (4.10.2) Mac OS 10.12.3 (16D32)

Example Request (without the API keys... not going to post those here).
````
https://api.untappd.com/v4/beer/info/452240?client_id=<client-id>&client_secret=<client-secret>
````

Example Response (snipped from within Postman, including screenshot):
````
"beer_name": "Cali CÃ¶ast",
"beer_description": "Cali CÃ¶ast is a continental mash-up. Brewed to celebrate flavorful light beers, this bright and playful ale blends the delicate peach and pear flavors of German Kolsch-Style ale yeast with fresh California grown barley.",
"beer_style": "KÃ¶lsch",
````
![image](https://cloud.githubusercontent.com/assets/2158627/23835573/fc4ee914-0726-11e7-8667-392fc28cad95.png)

Example Curl Command (same resource):
````
curl -v "https://api.untappd.com/v4/beer/info/452240?client_id=<client-id>&client_secret=<client-id"
````

Example Curl Response (snipped including headers and screenshot):
````
< Transfer-Encoding: chunked
< Content-Type: application/json
      "beer_name": "Cali Cöast",
      "beer_description": "Cali Cöast is a continental mash-up. Brewed to celebrate flavorful light beers, this bright and playful ale blends the delicate peach and pear flavors of German Kolsch-Style ale yeast with fresh California grown barley.",
      "beer_style": "Kölsch",
````
![image](https://cloud.githubusercontent.com/assets/2158627/23835571/e8c77b18-0726-11e7-82e4-6d486a422e2c.png)

Expected behavior is to properly show the `ö` character instead of what is currently being rendered `Ã¶`

**Comment by sdnts** [comment]
@tmack8001 Thanks for letting us know. We'll look into this :)

**Comment by SamvelRaja** [comment]
This issue has been fixed in our latest 6.1.4.
